app/models/project.rb

```diff
@@ -1,4 +1,16 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def experience
```

Comment:
You can use ActiveRecord here instead of ruby!

`self.contestants.average(:years_of_experience).to_f` and you can optionally choose to round

If you find yourself doing a lot of Ruby in a model method, there's usually a simpler AR solution!

Reply:
I will add, though, that encapsulating this logic into a model method was the right idea! Nice work. Anytime you add a method to a model though, we'd like to see a model test.